src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
```

Comment:
@layomia VS2022 automatically added these characters, so I think thats good :smile: But is that okay for this PR?

Reply:
It's better to have the only diff to this file be the updated exception message.